src/bin/grin.yml

```diff
@@ -16,6 +16,11 @@ subcommands:
   - server:
       about: Control the Grin server
       args:
+        - tui_enabled:
+            help: Run with or without TUI, bypassing config
+            short: t
+            long: tui_enabled
+            takes_value: true
```

Comment:
Would you consider to not take value? For me`grin -t true` looks awkward comparing to `grin -t`, also boolean value anyway modeled by presence/absence of the argument

Reply:
Yup, agreed here.